sql/modules/Roles.sql

```diff
@@ -195,16 +197,37 @@ GRANT SELECT ON location_class_to_entity_class TO PUBLIC;
 
 
 \echo BASE ROLES
-SELECT lsmb__create_role('base_user');
+SELECT lsmb__create_role(
+  'base_user',
+  $DOC$
+  Users need to be given this role in order to be granted access to the database schema which holds all LedgerSMB objects.
+  $DOC$
+  );
 SELECT lsmb__grant_schema('base_user', :'lsmb_schema');
 
 
 \echo BUDGETS
-SELECT lsmb__create_role('budget_enter');
-SELECT lsmb__create_role('budget_view');
-SELECT lsmb__create_role('budget_approve');
+SELECT lsmb__create_role('budget_enter',
+                         $DOC$
+                         This role allows creation and updating of budgets.
+                         $DOC$
+);
+SELECT lsmb__create_role('budget_view',
+                         $DOC$
+                         This role allows searching and viewing of budgets.
+                         $DOC$
+);
+SELECT lsmb__create_role('budget_approve',
+                         $DOC$
+                         This role allows searching, viewing and approving of budgets.
+                         $DOC$
+);
 SELECT lsmb__grant_role('budget_approve', 'budget_view');
-SELECT lsmb__create_role('budget_obsolete');
+SELECT lsmb__create_role('budget_obsolete',
+                         $DOC$
+                         This role allows searching, viewing and marking as obsolete of budgets.
```

Comment:
How about

This role allows searching and viewing of budgets as well as marking them as obsolete.

Reply:
Done.